x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -782,17 +775,19 @@ example below:
 
 [source, yaml]
 ------------------------------------------------------------
-xpack.security.authc.providers: [saml, basic]
+xpack.security.authc.providers:
+  saml.saml1:
+    order: 0
+    realm: "saml1"
+  basic.basic1:
+    order: 1
 ------------------------------------------------------------
 
-The order is important - this will _initiate_ SAML authentication for
-unauthenticated users, but will _accept_ basic authentication.
-
-If {kib} is configured in this way, then users who wish to login with a
-username and password, can do so by directly accessing the `/login` page in
-{kib}. This login will not use SAML credentials, and will rely on one of the
-other security realms within {es}. Only users who have a username and password
-for a configured {es} authentication realm will be able to login via this page.
+If {kib} is configured in this way, then users will be presented with the choice
+at the Login Selector screen whether to use username and password, and rely on one
```

Comment:
```suggestion
at the Login Selector screen. They can provide a username and password, rely on one
```

Reply:
Thanks! I kept `and` before `rely on` since these "other security realms" are used only when username and password are provided, but let me know if there is a better way to express this (in this sentence we're describing just two options - either SAML or not-SAML).

Reply:
Hmmm... maybe that will be more obvious if we change the order.  I'll provide a new suggestion.